docs/REST API/rest-api.md

```diff
@@ -9901,8 +9870,136 @@ Only one of `labels_include_any` or `labels_exclude_any` can be specified. If ne
 }
 ```
 
+### Patch software
+
+> **Experimental feature**. This feature is undergoing rapid improvement, which may result in breaking changes to the API or configuration surface. It is not recommended for use in automated workflows.
+_Available in Fleet Premium._
+
+Install software available for install on all hosts running an older version.
+
+`POST /api/v1/fleet/software/titles/:id/patch`
+
+#### Parameters
+
+| Name | Type | In | Description |
+| ---- | ---- | -- | ----------- |
+| software_title_id              | integer | path | **Required**. The ID of the software title. |
+| team_id | integer | query | **Required**. The team ID. |
+| enable_patch   | boolean | body | Whether or not patching is enabled or not for the specific software title. |
```

Comment:
@noahtalerman we currently have a mix of `enable_` and `_enabled` naming throughout the API. as much as possible, I'd like to standardize on `_enabled` as a suffix. (The main exception is if there are already surrounding booleans in an API's parameters that use `enable`; in those cases we should be consistent with what's nearby.)
```suggestion
| patch_enabled   | boolean | body | Whether or not patching is enabled or not for the specific software title. |
```